lib/util/get-scalar-outcome.ts

```diff
@@ -0,0 +1,22 @@
+import { Decimal } from "decimal.js";
+import { ZTG } from "../constants";
+import type { ScalarRangeType } from "@zeitgeistpm/sdk/dist/types";
+
+export const getScalarOutcome = (
```

Comment:
I would rename this to `formatScalarOutcome` since its not strictly a getter of the outcome but rather a display formatter.